ui/desktop/src/ChatWindow.tsx

```diff
@@ -13,6 +13,17 @@ import LoadingGoose from './components/LoadingGoose';
 import { ApiKeyWarning } from './components/ApiKeyWarning';
 import { askAi, getPromptTemplates } from './utils/askAI';
 import WingToWing, { Working } from './components/WingToWing';
+import { WelcomeScreen } from './components/WelcomeScreen';
+
+// Current version of the app - update this when you want to show the welcome screen again
+const CURRENT_VERSION = '1.0.0';
+
+// Get the last version from localStorage
+const getLastSeenVersion = () => localStorage.getItem('lastSeenVersion');
+const setLastSeenVersion = (version: string) => localStorage.setItem('lastSeenVersion', version);
+
+// Current version of the app - update this when you want to show the welcome screen again
+const CURRENT_VERSION = '1.0.0';
```

Comment:
This is duplicated - will be a hrd TS error

Reply:
yeah I don't think we should show version here - use about as MSC etc controls that

Reply:
actually can use this as just some indicator we want it to show fresh again or not - version number is not necessary